spaceship/lib/spaceship/connect_api/models/bundle_id_capability.rb

```diff
@@ -41,6 +41,21 @@ module Type
         MARZIPAN = "MARZIPAN" # Catalyst
       end
 
+      module Settings
+        ICLOUD_VERSION = "ICLOUD_VERSION"
+        DATA_PROTECTION_PERMISSION_LEVEL = "DATA_PROTECTION_PERMISSION_LEVEL"
+        APPLE_ID_AUTH_APP_CONSENT = "APPLE_ID_AUTH_APP_CONSENT"
+      end
+
+      module Options
+        XCODE_5 = "XCODE_5"
+        XCODE_6 = "XCODE_6"
+        COMPLETE_PROTECTION = "COMPLETE_PROTECTION"
+        PROTECTED_UNLESS_OPEN = "PROTECTED_UNLESS_OPEN"
+        PROTECTED_UNTIL_FIRST_USER_AUTH = "PROTECTED_UNTIL_FIRST_USER_AUTH"
+        PRIMARY_APP_CONSENT = "PRIMARY_APP_CONSENT"
+      end
```

Comment:
I read the document but it looks like `CapabilitySetting` is a quite complicated structure.
https://developer.apple.com/documentation/appstoreconnectapi/capabilitysetting

How did you compose that? Do we need to provide exactly all the properties described in the page?
If so, maybe we can help users to compose it by providing a plain Ruby object for it?

```ruby
class CapacitySetting
  attr_reader :key, :options # and etc..
  def initialize(key:, options:, ....)
end
```

Reply:
Hi @ainame 
Thank you for taking the time to explain all the stuff in your other comments!

Of all the endpoints I choose to implement, I found out that I choose to implement the most complicated one 😅

I have never used one of the capabilities that require configuration before, so I have no clue what all the parameters should be or which one is required.
I just added the possibility to add the settings, so that users that knows how to set it up, actually can do it.

But I will be happy to add an object for the `CapabilitySetting` if anyone can provide some information about thies parameters 😄

Reply:
I love this idea of adding a `CapabilitySetting` building 😊 I don’t think it has to be added on this PR but I will be making an update to `produce` soon that needs this so I’ll add this builder to that PR 💪 

I think leaving it out for now is perfectly fine for this PR but I do love the idea 😍